scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating STACKROX_ADMIN_PASSWORD"
+
+STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
+
+echo "Installing stackrox-central-services"
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="$STACKROX_ADMIN_PASSWORD"
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Generating an init bundle with shared secrets."
+
+kubectl -n stackrox exec -it deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 0>/dev/null 1> stackrox-init-bundle.yaml 2>/dev/null
```

Comment:
1. What's the point to redirect file descriptor `0` to `/dev/null`??
2. Also, you probably don't need `-it` command-line options on `kubectl exec`.
3. I don't think there's going to be a problem if you omit `2>/dev/null`. Is there a problem for you? Can you share output of that?

Reply:
1. If I leave out `0>/dev/null`I get `Error: INSTALLATION FAILED: failed to parse stackrox-init-bundle.yaml: error converting YAML to JSON: yaml: control characters are not allowed` and inside my stackrox-init-bundle.yaml file there are lines such as `[94mINFO:	Successfully generated new init bundle.` making it impossible to parse.
2. I'm actually not sure what the `-it` is for exactly, I added it on a suggestion while debugging the above problem. Running the command without it works fine so I can leave it out.
3. Omitting `2>/dev/null` it still worked, I had previously added both `0>/dev/null` and `2>/dev/null`, saw that it now worked and left it like that. I'll remove this one.

If I'm understanding the problem correctly it's that the StdErr stream is being sent to the yaml file instead of the terminal output, possible because roxctl is being run within `kbectl -n stackrox exec -it deploy/central --`. I had been discussing this with @vikin91 previously, do you know a way to pipe `0>` so that the output appears in the terminal running the script Misha?

Reply:
Re 1: sorry, your explanation does not make sense to me. I don't see how file descriptor `0` may be related to the output. Try googling "bash I/O Redirection", there's an article on https://tldp.org that explains `0`, `1` and `2`.

Reply:
Re 2: the information is all at your fingertips, again.
```sh
$ kubectl help exec
...
  -i, ...
  -t, ...
...
```

Reply:
Well regarding 1: I'm not sure why it works this way either. After reading that article you mentioned I/O redirection is a lot clearer to me and the behavior here is strange. Regarding 2: I should put more time into understanding bash rather than just calling it done when it's working. Next time I'll look into what all the options and flags I'm using do and learn some more rather then move on to the next thing too quickly.

Reply:
I checked out your branch, modified the command as following
```sh
kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
  --password "${STACKROX_ADMIN_PASSWORD}" \
  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
```
and it works perfectly fine for me.
If you run the same, what's the error that you see? What's the content of `stackrox-init-bundle.yaml`?

Reply:
It actually works now. It turns out that removing the -it option helped. Attached is the output `stackrox-init-bundle.yaml`if I run it with the -it option, like so:
```
kubectl -n stackrox exec -it deploy/central -- roxctl --insecure-skip-tls-verify \
  --password "${STACKROX_ADMIN_PASSWORD}" \
  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
  ```
 
Why it was fixed in that case by adding `0>/dev/null` is beyond my understanding though.

[stackrox-init-bundle.txt](https://github.com/stackrox/stackrox/files/9250641/stackrox-init-bundle.txt)